python/tvm/ir/expr.py

```diff
@@ -62,6 +62,17 @@ def shape(self):
         """
         return _ffi_api.RelayExprShape(self)
 
+    @property
+    def struct_info(self):
```

Comment:
Just out of curiosity: do we have the convention to type-annotate the property-decorated methods?
```suggestion
    def struct_info(self) -> "tvm.relax.StructInfo":
```